### ISSUE TITLE
Pkg.build("ImageMagick") version 0.2.1 on OS X leads to segmentation fault

### ISSUE THREAD

**Issue by BoundaryValueProblems** [JuliaIO/ImageMagick.jl#74]
I can neither update nor build ImageMagick 0.2.1 on OS X (with julia 0.5.0) properly.
I got the following errors:
```julia
julia> Pkg.build("ImageMagick")
INFO: Building Homebrew
Already up-to-date.
Uninstalling imagemagick@6... (1,467 files, 22.3M)
==> Installing imagemagick@6 from staticfloat/juliatranslated
==> Downloading https://juliabottles.s3.amazonaws.com/imagemagick@6-6.9.7-3.el_c
Already downloaded: /Users/xxx/Library/Caches/Homebrew.jl/imagemagick@6-6.9.7-3.el_capitan.bottle.tar.gz
==> Pouring imagemagick@6-6.9.7-3.el_capitan.bottle.tar.gz
🍺  /Users/xxx/.julia/v0.5/Homebrew/deps/usr/Cellar/imagemagick@6/6.9.7-3: 1,467 files, 22.3M
Warning: Already linked: /Users/xxx/.julia/v0.5/Homebrew/deps/usr/Cellar/imagemagick@6/6.9.7-3
INFO: Building ImageMagick

signal (11): Segmentation fault: 11
while loading /Users/xxx/.julia/v0.5/ImageMagick/deps/build.jl, in expression starting on line 122
MagickQueryConfigureOption at /Users/xxx/.julia/v0.5/Homebrew/deps/usr/lib/libMagickWand-6.Q16.dylib (unknown line)
anonymous at ./<missing> (unknown line)
unknown function (ip: 0x316010dee)
...
```
This is Mac OS X specific problem since I didn't have any problem building it on my Linux machine and on my Windows machine.
Please help!
BVPs

**Comment by bjarthur** [comment]
dup of https://github.com/JuliaPackaging/Homebrew.jl/issues/175#issuecomment-279494001

**Comment by BoundaryValueProblems** [comment]
@bjarthur , I don't think this problem has been resolved yet although JuliaPackaging/Homebrew.jl#175 (comment) was closed. I still get segmentation fault when I try to update/build ImageMagick.jl on Mac OS X!

**Comment by SimonDanisch** [comment]
I can also reproduce the segfault on 0.2.0 -> master ...

**Comment by samuelpowell** [comment]
Following @Allardvm's instructions (https://github.com/JuliaPackaging/Homebrew.jl/issues/175#issuecomment-288381959) fixed this for me.

**Comment by mkborregaard** [comment]
@Allardvm may have solved this issue (see the above-linked issue) with a PR on the way :tada:

**Comment by timholy** [comment]
Presumably closed by #90. Feel free to reopen or comment if not.
